src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+header,
+h1,
+ul {
+  margin: 0;
+}
+
+/* html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+} */
+
+.styleHeader {
+  width: 1200px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  text-transform: uppercase;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  cursor: pointer;
```

Comment:
why do you need this on `nav`?

Reply:
fix it https://github.com/mate-academy/layout_moyo-header/pull/2626/files#r1096597517